include/tvm/relay/vm/vm.h

```diff
@@ -131,65 +164,62 @@ void VMFunctionPrint(const VMFunction& vm_func);
  */
 struct VMFrame {
     size_t pc;
-    size_t bp;
-    size_t sp;
     size_t func_index;
     size_t args;
     const Instruction* code;
 
-    VMFrame(size_t pc, size_t bp, size_t sp, size_t func_index, size_t args, const Instruction* code)
-      : pc(pc), bp(bp), sp(sp), func_index(func_index), args(args), code(code) {}
+    // Points to the offset of this function's register file start inside VM's register stack
+    size_t register_stack_start;
+
+    // map from virtual register to memory slot
+    std::unordered_map<VirtualRegisterNum, SlotNum> register_file_map;
```

Comment:
This is my point, we don't need this we can just number virtual slots from starting at the base pointer, 0, 1, 2, etc.

For example let's assume we have `99` graph vars, we will assign to `99` slots without needing to do register allocation.

Reply:
@jroesch I think I addressed your comment in latest revision :-)